api/channel.go

```diff
@@ -953,6 +954,37 @@ func getChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 
 }
 
+func getChannelByName(c *Context, w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+	channelname := params["channel_name"]
+
+	cchan := Srv.Store.Channel().GetByName(c.TeamId, channelname)
+
+	if cresult := <-cchan; cresult.Err != nil {
+		c.Err = cresult.Err
+		return
+	} else {
+		data := &model.Channel{}
```

Comment:
I talked in the developers channel and asking if this api should return the members as well, was told to me for this api we just need return the channel and not the channel members.
@coreyhulen please let me know your thoughts if need return the members as well I agree with you this is similar to the getchannel